evm_loader/program/Cargo.toml

```diff
@@ -34,7 +34,7 @@ test-bpf = []
 custom-heap = []
 default = ["custom-heap"]
 
-tracing = []
+library = ["maybe-async", "async-trait"]
```

Comment:
evm_loader crate can be compiled as 2 different kind of libraries.
1. As x86 tracer library, which is used by cli, api, etc
2. As BPF library, which could be used by an another Solana contract.

Reply:
Yes, this is why `library` feature is not enabled by default. It is enabled only when `evm_loader` crate is used as a library.